packages/config-plugins/src/ios/Paths.ts

```diff
@@ -5,7 +5,7 @@ import * as path from 'path';
 import { UnexpectedError } from '../utils/errors';
 import * as WarningAggregator from '../utils/warnings';
 
-const ignoredPaths = ['**/@(Carthage|Pods|node_modules)/**'];
+const ignoredPaths = ['**/@(Carthage|Pods|vendor|node_modules)/**'];
```

Comment:
We use this pattern in a few places, can you check to see if they need updated too.

Reply:
Thanks. I confirmed the patterns needed to be updated in other places as well (`eas build:configure` failed for the same reason as well). I updated other files.